README.md

```diff
@@ -117,9 +88,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://last-3f0w.onrender.com/users/1/groups)
```

Comment:
- [ ] I noticed that your video link is too long around 7 minutes . The requirement for the video length  is 5 minutes maximum . Please fix it make sure that the new video follows all the videos project requirement :+1: